packages/calypso-e2e/package.json

```diff
@@ -15,12 +15,11 @@
 	],
 	"license": "GPL-2.0-or-later",
 	"dependencies": {
+		"@types/config": "^0.0.38",
```

Comment:
We should be able to move this back to dev dependencies now that yarn install is fixed

Reply:
Ok, will do that.

Reply:
Pushed subsequent commit that reverts this change.